_migrations/migration-phases/backfill.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: default
+title: Backfill
+nav_order: 90
+parent: Migration phases
+---
+
+# Backfill
+
+After the [metadata]({{site.url}}{{site.baseurl}}/migrations/migration-phases/migrating-metadata/) for your cluster has been migrated, you can use capture proxy data replication and snapshots to backfill your data into the next cluster.
+
+## Capture proxy data replication
```

Comment:
While this was associated with the previous section, these details are incredibly confusing for users focused on the backfill-only case.  I would keep that `Capture Proxy Data Replication` in its own section that is referenced for when a user is running a capture proxy & backfill migration at the same time.

Reply:
This covers everything from lines 12-75

Reply:
I would leave here but make it clearer that this is only needed if a user is interested in capturing live traffic. The reason it should be here is because it makes this page consistent with the steps outline within our repo's primary page and the Migration Assistant landing page.

Reply:
Well @Naarcha-AWS looks like it is up to you - I'm fine either way :D

Reply:
To be clear, what are the two scenarios? As I understand it:

1. Using ALB with a capture proxy for backfill and NLB. 
2. Only using backfill.

Explicitly, what's unclear to me is why you would us one scenario over the other. Any insight on this @peternied or @sumobrian?

Reply:
There is flexibility in how the MA is used; it for 1) backfill (via snapshot + rfs), 2) capture and replay (via proxy + kafka + replayer), or a 3) combination of the two.

My comment was coming more from scenario 1, where capture and replay isn't a consideration.  But that is a simplification of the scenarios customers will use.

For scenario 3 that @sumobrian calls out, making sure the capture and reply system is working correctly is critical to prevent data lost between the snapshot and the start of the requests captured.  This makes covering these configurations important for the workflow.

As this is the first official release of this documentation we can always revisit if we come up with a different presentation model that better fits customer expectations.

Reply:
Lets address it at a later date.

Reply:
I'll add a sentence that this is mainly for capturing live traffic.